packages/sandbox/src/file_watching_sandbox.ts

```diff
@@ -325,9 +325,9 @@ export class FileWatchingSandbox extends EventEmitter implements Sandbox {
       if (
         e &&
         typeof e === 'object' &&
-        'message' in e &&
-        typeof e.message === 'string' &&
-        e.message.includes('does not exist')
+        'name' in e &&
+        typeof e.name === 'string' &&
+        e.name === 'ParameterNotFound'
```

Comment:
see https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-client-ssm/Class/ParameterNotFound/

Reply:
should we instead do `if (e instance of ParameterNotFound)`?

Reply:
looks like we should in aws-sdk v3.53.0+ https://aws.amazon.com/blogs/developer/service-error-handling-modular-aws-sdk-js/

Reply:
@Amplifiyer @awsluja done.